### ISSUE TITLE
Thresholds, still confused..

### ISSUE THREAD

**Issue by rranke** [Tucsky/SignificantTrades#29]
Why is 
```
huobi 6784 sats 3.8168141 56K
```
showing animated-background?

Console:
```
JSON.stringify(JSON.parse(localStorage.getItem('settings')).thresholds)
"[{\"amount\":0.5,\"buyColor\":\"rgba(76,175,80,.33)\",\"sellColor\":\"rgba(229,115,115,.33)\"},{\"amount\":2.5,\"buyColor\":\"rgb(91,130,48)\",\"sellColor\":\"rgb(224,91,82)\"},{\"amount\":5,\"gif\":\"cash\",\"buyColor\":\"rgb(156,204,101)\",\"sellColor\":\"rgb(244,67,54)\"},{\"amount\":10,\"gif\":\"explosion\",\"buyColor\":\"rgb(255,160,0)\",\"sellColor\":\"rgb(233,30,99)\"}]"
```

![Bildschirmfoto vom 2019-04-09 16:11:27](https://user-images.githubusercontent.com/5783257/55807769-10f3cd00-5ae3-11e9-9a1b-ab77720e0df3.png)

**Comment by Tucsky** [comment]
I think it can happen when you swap 1 threshold with another, a simple refresh might fix it tho...
Ill push a fix tonight

**Comment by Tucsky** [comment]
Actually

As you might already know, each exchanges have a settings for threshold ajustement
https://github.com/Tucsky/SignificantTrades/blob/master/src/services/store.js#L21
Huobi for instance is set to 66% by default meaning if you want to show gif at 5 BTC and a 3.81BTC huobi trade show up, the gif will be visible because its huobi and 5*.66=3.3 :)

**Comment by rranke** [comment]
"As you might already know, each exchanges have a settings for threshold"

Ahhh, okay, did not notice that.
That should clear up the confusion!
I will check that out, thanks.

**Comment by Tucsky** [comment]
Fixed d28b681bc739966d20898b20ad53c7cfcfe39dbe
